src/OptOut.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\OptOut.
+ */
+
+namespace Netzstrategen\CoreStandards;
+
+class OptOut {
```

Comment:
This appears to be all about Google Analytics, so why don't we call it like that?

Simply `GoogleAnalytics` or if you think that's too unspecific then `GoogleAnalyticsOptOut`

Reply:
@sun Changed it to `TrackingOptOut` instead of `GoogleAnalyticsOptOut` to left it open to cover other tracking systems in the future (see "nice to have features" in the task summary).